qa/tasks/vstart_runner.py

```diff
@@ -271,8 +271,10 @@ def put_file(self, src, dst, sudo=False):
         try:
             shutil.copy(src, dst)
         except exception as e:
-            if sys.version_info.major < 3 and e.message.find('are the same '
-               'file') != -1:
+            if sys.version_info.major < 3:
+                if e.message.find('are the same file') != -1:
+                    return
+            else:
                 return
             raise e
```

Comment:
```suggestion
            if sys.version_info.major < 3:
                if  e.message.find('are the same file') != -1:
                    return
                raise e
```

Reply:
And what about when Python version is `sys.version_info.major >= 3`?